docs/src/TextArea.doc.js

```diff
@@ -234,6 +240,71 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
```

Comment:
One thing I noticed: in your Tyepahead example, after deleting a tag, focus is back on the input, but that doesn't seem to be the case here in Textarea. After deleting a tag in text area, focus is lost.

Reply:
Same behavior in `Textfield`.